routes/products.js

```diff
@@ -0,0 +1,17 @@
+const { getProductById } = require('../db');
+
+const express = require('express');
+const productsRouter = express.Router();
+
+productsRouter.get('/products/:productId', async (req, res, next) => {
```

Comment:
change `/products` to `/product`

Reply:
rename index.js file to client.js. create a new index.js where you are going to module.export products